malevich/my_flows.py

```diff
@@ -9,56 +9,54 @@
 ProcFunReturn = TypeVar("ProcFunReturn")
 
 class FlowFunctionStub(Generic[ProcFunArgs, ProcFunReturn]):
-    def __init__(self, fn, mapping, reverse_id, version) -> None:
-        self.mapping: dict = mapping
-        self.reverse_id = reverse_id
-        self.flow_version = version
+    def __init__(self, fn, reverse_id) -> None:
         self._fn_ = fn
         self.__call__ = self._fn_call
+        self.reverse_id = reverse_id
 
     def _fn_call(self, *args, **kwargs) -> list[SpaceCollectionResult]:
         parameters = list(inspect.signature(self._fn_).parameters.values())
         mapped_kwargs = {
-            self.mapping.get(k): v
+            k: v
             for k, v in kwargs.items()
         }
         varnames = [
             p.name for p in parameters
             if p.kind == inspect.Parameter.POSITIONAL_OR_KEYWORD
         ]
-        for arg, key in zip(islice(args, 0, len(varnames)), self.mapping.values()):
+        for arg, key in zip(islice(args, 0, len(varnames)), varnames):
```

Comment:
use `def _fn_call(self, version, branch, *, task_policy=..., **kwargs)`. Then your kwargs will contain only collections, and no need for all of these fors and etc.